src/components/Table/Table.jsx

```diff
@@ -164,12 +165,21 @@ class Table extends React.Component {
                       }
 
                       let fieldEl = null;
-                      const bars = relativeBars || percentBars || percentBarsWithValue;
+                      const bars = relativeBars || relativeNegativeBars || percentBars || percentBarsWithValue;
                       if (bars) {
                         const altValue = typeof bars === 'function' && percentBarsWithValue ? bars(row) : null;
                         let valEl = null;
                         let barPercentValue = 0;
-                        if (relativeBars) {
+                        if (relativeNegativeBars) {
```

Comment:
Instead of redoing all this logic, I think we should just pass a single property down called "invertedScaling" and flip the barPercentValue